velox/functions/prestosql/tests/UuidFunctionsTest.cpp

```diff
@@ -84,6 +84,32 @@ TEST_F(UuidFunctionsTest, castAsVarchar) {
   ASSERT_EQ(size, uniqueUuids.size());
 }
 
+TEST_F(UuidFunctionsTest, castAsVarbinary) {
```

Comment:
@xin-zhang2 : Write a castRoundTrip function with VARBINARY as well. See the castRoundTrip function with VARCHAR right below this function.